tailwind.config.js

```diff
@@ -1,6 +1,26 @@
 module.exports = {
   theme: {
-    extend: {}
+    textColor: theme => ({
```

Comment:
that's cool =), did you go through thiswith ben ? might be interesting for him

Reply:
yup he's aware

Reply:
yes i saw this